frontend/public/static/locale/en-US/translate.ftl

```diff
@@ -475,9 +475,9 @@ user-UserMenu--download-tm = <glyph></glyph>Download Translation Memory
 user-UserMenu--download-translations = <glyph></glyph>Download Translations
 user-UserMenu--upload-translations = <glyph></glyph>Upload Translations
 
-user-UserMenu--top-contributors = <glyph></glyph>Top Contributors
-user-UserMenu--machinery = <glyph></glyph>Machinery
```

Comment:
Sorry for not noticing this before - could you please remove these two entries also from the fr file?
https://github.com/mozilla/pontoon/blob/2fb5ea62fb474ef219c926ae41b91e6c64be5bd7/frontend/public/static/locale/fr/translate.ftl#L302

Reply:
How about for Hack it on Github and Give feedback? They are missing in that file. Should I add them? @mathjazz

Reply:
That's OK - the fr and es files are not fully localized yet. We have to fix a few more [localizability bugs](https://wiki.mozilla.org/L10n:Pontoon#Translate.Next_Followup) and then we'll start localizing Pontoon itself on pontoon.mozilla.org.